Automattic/wp-calypso
# Remove mention of lazy loading images from comparison page

## Proposed Changes

* Remove mention of lazy loading images from comparison page
* Remove something called a tour about lazy loading images. I cannot recall seeing this before and I could not track down a place it is currently being used, but removed it anyway

## Testing Instructions

1. Use the Calypso live link or checkout this branch locally
2. Go to `/features/comparison` on jetpack cloud and make sure the Lazy Loading row is gone
![image](https://github.com/Automattic/wp-calypso/assets/65001528/eee7ca49-e7de-4b28-8702-119b7cd4a350)

These are the only instances I could find on Calypso, you could also check to see if I missed any. I searched for "Lazy Loading", "Lazy loading" and "lazy loading". 

## Pre-merge Checklist
